apps/client/src/features/rundown/event-block/composite/EventBlockReporter.tsx

```diff
@@ -0,0 +1,93 @@
+import { Tooltip } from '@chakra-ui/react';
+import { OntimeEvent, ReportData } from 'ontime-types';
+import { isPlaybackActive, MILLIS_PER_MINUTE, MILLIS_PER_SECOND } from 'ontime-utils';
+
+import { usePlayback, useTimeUntil } from '../../../../common/hooks/useSocket';
+import useReport from '../../../../common/hooks-query/useReport';
+import useRundown from '../../../../common/hooks-query/useRundown';
+import { cx } from '../../../../common/utils/styleUtils';
+import { formatDuration, getTimeToStart } from '../../../../common/utils/time';
+import { tooltipDelayFast } from '../../../../ontimeConfig';
+
+import style from './EventBlockReporter.module.scss';
+
+interface EventBlockReporterProps {
+  id: string;
+  timeStart: number;
+  isPast: boolean;
+  isLoaded: boolean;
+  className: string;
+}
+
+function EventReport(props: { id: string; className: string }) {
+  const { id, className } = props;
+  const { data } = useReport();
+  const {
+    data: { rundown },
+  } = useRundown();
+
+  const report: ReportData | undefined = data[id];
+
+  if (report && report.startAt !== null && report.endAt !== null) {
+    const thisEvent = rundown[id] as OntimeEvent;
+    const expectedDuration = thisEvent.duration;
+    const actualDuration = report.endAt - report.startAt;
+    const overUnder = actualDuration - expectedDuration;
+    if (Math.abs(overUnder) > MILLIS_PER_SECOND) {
+      const isNegative = overUnder < 0;
+      const absOverUnder = Math.abs(overUnder);
+
+      //show seconds if the amount is less than 2 minutes as it could then represent up to 50% of the actual value?
+      const reportDisplay = `${isNegative ? '-' : '+'}${formatDuration(absOverUnder, absOverUnder > MILLIS_PER_MINUTE * 2)}`;
+
+      return (
+        <Tooltip label='Offset from lats run' openDelay={tooltipDelayFast}>
+          <div className={cx([style.chip, className, isNegative ? style.under : style.over])}>{reportDisplay}</div>
+        </Tooltip>
+      );
+    }
+  }
+  return null;
+}
+
+function EventUntil(props: { className: string; timeStart: number }) {
+  const { timeStart, className } = props;
+  const { clock, offset } = useTimeUntil();
```

Comment:
it would be good to try and stabilise this value to match our rules

something like (I didnt test it)
```ts
function useLessChangingTimeUntil(timeStart: number) {
  const { clock, offset } = useTimeUntil();

  const lazyTimeToStart = useMemo(() => {
    // we dont show anything if there is no offset
    if (offset === null) {
      return;
    }

    const timeToStart = getTimeToStart(clock, timeStart, 0, offset);
    // if the event is due, we dont need need the accurate value
    if (timeToStart <= MILLIS_PER_SECOND) {
      return 0;
    }

    // if the timer is over 2 minutes, we just show minutes
    if (timeToStart > MILLIS_PER_MINUTE * 2) {
      return formatDuration(timeToStart);
    }

    // at this point, timeToStart is positive and larger than 2 minutes
    return formatDuration(timeToStart, false);
  }, [clock, offset, timeStart]);

  return lazyTimeToStart
}
```